### DISCUSSION TITLE
Keep Qtile profile or remove it from the Dotfiles for 2.9.5

### DISCUSSION THREAD

**Discussion by mylinuxforwork** [mylinuxforwork/dotfiles#79] [Polls]
I am currently thinking about removing the Qtile profile from the ML4W Dotfiles. Actually Qtile was for me a backup at the beginning of my Hyprland experiments but haven't logged in to Qtile since months.

**Comment by kerbyyy** [comment]
Same case, never had a reason to use or install Qtile. One of the main reason why I'm using this dotfiles is because of how good the configs and features made for Hyprland. I think focusing on Hyprland will make maintaining and development much easier, especially if we're now considering other distros as well.

**Reply by mylinuxforwork** [reply]
Thanks for your feedback.

**Comment by eljejer** [comment]
It is indeed amazing, for my part anyway, what you have created with your dotfiles for Hyperland Stephan. I have not and will not be using Qtile.

**Comment by bknightInfo** [comment]
Hi Stephan,

I think most people come here for your Hyprland dots but maybe you could move Qtile dots to a qtile-starter repro.

**Reply by mylinuxforwork** [reply]
That is what I am currently think about.

**Comment by codeRSH** [comment]
Hello,

Are there any plans to bring back Qtile dotfiles, may be as a separate repo ?

While I  also primarily plan to use Hyprland dotfiles, but with Qtile improving Wayland support, I am curious to try Qtile as well!

**Reply by mylinuxforwork** [reply]
Please check https://github.com/mylinuxforwork/qtile-starter 
I am still fine tuning the config.
But you can install it already with the Dotfiles Installer.

**Reply by codeRSH** [reply]
Oh great! I somehow missed this repo . But I am on Fedora 42. Will the dotfiles installer automatically take care of the installation of relevant fedora packages ?

**Reply by mylinuxforwork** [reply]
> Oh great! I somehow missed this repo . But I am on Fedora 42. Will the dotfiles installer automatically take care of the installation of relevant fedora packages ?

No. Not yet. You have to install the dependencies manually. And then run the Dotfiles Installer afterwards.
